src/test/java/it/cnr/isti/hpc/wikipedia/article/en/ArticleTest.java

```diff
@@ -185,7 +185,7 @@ public void testAnnotationsInLists() throws IOException {
 			}
 		}
 
-		assertThat(uris, hasItems("Yū_Hayami", "Takumi_Hayami", "Hayami_District,_Ōita", "Mokomichi_Hayami"));
+		assertThat(uris, hasItems("Yū_Hayami", "Takumi_Hayami", "Hayami_District,_Ōita", "Mokomichi_Hayami", "H2O:_Footprints_in_the_Sand"));
```

Comment:
No.

That item has been there all the time, I just did not check its extraction in the test, cause I was 100% sure that it was failing.
So with the fixest that this PR introduces, this URI should be def extracted, thats why I added it to the test.